apps/site/src/app/(home)/sections/FAQ/getQuestions.ts

```diff
@@ -0,0 +1,43 @@
+import { z } from "zod";
+import { cache } from "react";
+// import { client } from "@/lib/sanity/client";
```

Comment:
Nitpick: This commented line isn't necessary here because you import it in the very next line.